web/cobrands/shropshire/assets.js

```diff
@@ -0,0 +1,189 @@
+(function(){
+
+if (!fixmystreet.maps) {
+    return;
+}
+
+var wfs_host = fixmystreet.staging ? 'tilma.staging.mysociety.org' : 'tilma.mysociety.org';
+var tilma_url = "https://" + wfs_host + "/mapserver/shropshire";
+
+var defaults = {
+    http_options: {
+        url: tilma_url,
+        params: {
+            SERVICE: "WFS",
+            VERSION: "1.1.0",
+            REQUEST: "GetFeature",
+            SRSNAME: "urn:ogc:def:crs:EPSG::27700"
+        }
+    },
+    asset_type: 'spot',
+    asset_id_field: 'central_as',
+    attributes: {
+        central_asset_id: 'central_as',
+        site_code: 'site_code'
+    },
+    geometryName: 'msGeometry',
+    srsName: "EPSG:27700",
+    body: "Shropshire Council",
+    strategy_class: OpenLayers.Strategy.FixMyStreet
+};
+
+var streetlight_stylemap = new OpenLayers.StyleMap({
+    'default': fixmystreet.assets.style_default,
+    'hover': fixmystreet.assets.style_default_hover,
+    'select': fixmystreet.assets.construct_named_select_style("${feature_id}")
+  });
+
+var labeled_defaults = $.extend(true, {}, defaults, {
+    select_action: true,
+    feature_code: 'feature_id',
+    stylemap: streetlight_stylemap,
+    construct_asset_name: function(id) {
+        var code = id.replace(/[O0-9]+[A-Z]*/g, '');
+        return {id: id, name: streetlight_code_to_type[code] || 'street light'};
+    },
+    actions: {
+        asset_found: fixmystreet.assets.named_select_action_found,
+        asset_not_found: fixmystreet.assets.named_select_action_not_found
+    }
+});
+
+var highways_style = new OpenLayers.Style({
+    fill: false,
+    strokeColor: "#5555FF",
+    strokeOpacity: 0.1,
+    strokeWidth: 7
+});
+
+fixmystreet.assets.add(defaults, {
+    http_options: {
+        params: {
+            TYPENAME: "Street_Gazeteer"
+        }
+    },
+    stylemap: new OpenLayers.StyleMap({
+         'default': highways_style
+     }),
+    always_visible: true,
+    non_interactive: true,
+    road: true,
+    asset_item: 'road',
+    asset_type: 'road',
+    all_categories: true,
+});
```

Comment:
No, asset_item and asset_type are used in messages such as "Zoom in to pick an asset_item" (and asset_type is used in the CSS class). By attribute, I meant a `usrn:` attribute on the object instead of `road: true` (which is for calling found/not found code on road layers, which you don't have here), see examples in other cobrand files and `updateUSRNField` in fixmystreet/assets.js for how that attribute is used.